Project.toml

```diff
@@ -33,6 +34,7 @@ julia = "1.6"
 
 [extras]
 FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
+Functors = "d9f16b24-f501-4c13-a1f2-28368ffc5196"
```

Comment:
where is this used? you haven't added it to test nor imported it anywhere

Reply:
doh, it's used in a test...

Reply:
need to add it to `test` under `targets`!